apps/prairielearn/package.json

```diff
@@ -91,7 +91,7 @@
     "cookie-parser": "^1.4.7",
     "cropperjs": "^2.0.1",
     "crypto-js": "^4.2.0",
-    "csv-parse": "^5.6.0",
+    "csv-parse": "^6.1.0",
```

Comment:
The only breaking changes were properties that we don't use: https://github.com/adaltas/node-csv/blob/master/packages/csv-parse/CHANGELOG.md